File: Source/JapaneseLanguageTools.ReactSpa/source/models/blobs/responses/GetBlobExpirationTimeResponse.Model.ts
import BlobMetadataModel from "@/models/blobs/BlobMetadata.Model";
import GetBlobExpirationTimeRequestModel from "@/models/blobs/requests/GetBlobExpirationTimeRequest.Model";

export default interface GetBlobExpirationTimeResponseModel {
  blobMetadata: BlobMetadataModel;

  request: GetBlobExpirationTimeRequestModel;
}

File: Source/JapaneseLanguageTools.Contracts/Models/Blobs/Responses/GetBlobExpirationTimeResponse.Model.cs
using System;

using JapaneseLanguageTools.Contracts.Models.Blobs.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Blobs.Responses;

public class GetBlobExpirationTimeResponseModel
{
    public required DateTimeOffset? BlobExpirationTime { get; set; }

    public required GetBlobExpirationTimeRequestModel Request { get; set; }
}
